concent_api/utils/helpers.py

```diff
@@ -2,16 +2,19 @@
 import base64
```

Comment:
Commit name: `Fix for implementation of maximum_download_time and subtask_verification_time`

Please be more specific in commit names. This should really tell us what you're fixing without having to look into the code. What was broken before? How was it fixed?

It's not really a fix to be honest. You're just adding support for `settings.CUSTOM_PROTOCOL_TIMES`. And maybe you're changing something in the implementation but it's hard to tell at a glance.

Reply:
:+1: Ok, I will pay more attention to this next time.